packages/website/src/pages/docs/Guides/Language.mdx

```diff
@@ -8,39 +8,156 @@ import { SquiggleEditor } from "@quri/squiggle-components";
 
 Squiggle supports some simple types and language features.
 
-## Numbers
+## Basic types
 
-<SquiggleEditor defaultCode="4.32" />
+### Numbers
 
-## Distributions
+There's no difference between floats and integers in Squiggle.
 
-There are several ways of easily entering distributions. See the [documentation](/docs/Api/Dist/) on distributions for a complete API.
+Numbers support a few scientific notation suffixes.
+
+| suffix | multiplier |
+| ------ | ---------- |
+| n      | 10^-9      |
+| m      | 10^-3      |
+| k      | 10^3       |
+| M      | 10^6       |
+| B,G    | 10^9       |
+| T      | 10^12      |
+| P      | 10^15      |
+
+<SquiggleEditor defaultCode={`n = 4.32
+kilo = 4.32k
+micro = 4.32m`} />
+
+### Booleans
+
+Can be `true` or `false`.
+
+<SquiggleEditor defaultCode={`t = true
+f = false`} />
+
+### Strings
+
+<SquiggleEditor defaultCode={`s = "Double-quoted"
+s2 = 'Single-quoted'`} />
+
+### Distributions
+
+Distributions are first-class citizens in Squiggle. There are several ways to create them:
+
+- [to](/docs/Guides/DistributionCreation#to) operator.
+- [constructors for many kinds of distributions](/docs/Api/Dist#distribution-creation): `normal`, `lognormal`, `uniform`, and so on.
+- distributions can be combined with [mixture](/docs/Api/Dist#mixture) function or with [arithmetic operations](/Api/Dist#regular-arithmetic-operations).
+
+ See the [Dist module documentation](/docs/Api/Dist) for a complete API.
 
 <SquiggleEditor
-  defaultCode={`a = normal(4,2)
-b = 30 to 50
-c = lognormal({mean:90, stdev: 7})
-d = mixture(a,b,c, [.3, .3, .4])
+  defaultCode={`a = 30 to 50
+b = normal(4, 2)
+c = lognormal({ mean: 90, stdev: 7 })
+d = mixture(a, b, c, [.3, .3, .4])
 d`}
 />
 
-## Lists
+## Container types
+
+### Lists
+
+Squiggle lists can contain items of any type, similar to those in Python.
+
+You can access individual list elements with `[number]` notation, starting from `0`.
 
-Squiggle lists can accept items of any type, similar to those in Python. [API](/docs/Api/List).
+For common operations on lists, refer to the [List module API](/docs/Api/List).
 
 <SquiggleEditor
-  defaultCode={`[beta(1,10), 4, isNormalized(SampleSet.fromDist(1 to 2))]`}
+  defaultCode={`myList = [1, "hello", 3 to 5, ["foo", "bar"]]
+first = myList[0] // 1
+bar = myList[3][1] // "bar"`}
 />
 
-## Dictionaries
+### Dictionaries
 
-Squiggle dictionaries work similarly to Python dictionaries. [API](/docs/Api/Dictionary).
+Squiggle dictionaries work similarly to Python dictionaries or Javascript objects.
+
+For common operations on dictionaries, refer to the [Dict module API](/docs/Api/Dictionary).
 
 <SquiggleEditor
   defaultCode={`d = {dist: triangular(0, 1, 2), weight: 0.25}
 d.dist`}
 />
 
+## Other types
+
+Other value types that Squiggle has include:
+- [Functions](#functions)
+- [Plot objects](/docs/Api/Plot)
+- [Scales](/docs/Api/Plot#scales)
+- [Domains](#parameter-annotations)
```

Comment:
Add tables?

Reply:
There's nowhere to link yet, will add later